website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -23,15 +22,27 @@ part of the process to [Integrating Kubebuilder and Operator SDK][integration-do
 
 ### What is new
 
-Scaffolded projects now use:
+Projects are now scaffold using:
 
 - [kustomize][kustomize] to manage Kubernetes resources needed to deploy your operator
 - A `Makefile` with helpful targets for build, test, and deployment, and to give you flexibility to tailor things to your project's needs
 - Updated metrics configuration using [kube-auth-proxy][kube-auth-proxy], a `--metrics-addr` flag, and [kustomize][kustomize]-based deployment of a Kubernetes `Service` and prometheus operator `ServiceMonitor`
+- A preliminary support for plugins. For more info see the [extensible CLI and scaffolding plugins design document][plugins-phase1-design-doc]
+- A PROJECT file which stores more information about the resources in use, to enable plugins make useful decisions while scaffolding
+- Liveness and Readiness probes using [`healthz.Ping`][healthz-ping].
 
 ## How to migrate
 
-The easy migration path is to a project from the scratch and let the tool scaffold the new layout. Then, add your customizations and implementations. See below for an example.
+The easy migration path is to create a new project from the scratch and let the tool scaffold the files properly and then,
+just replace with your customizations and implementations. Following an example.
+
+## Migration Steps
+
+The most straightforward migration path is to:
+1. Create a new project from scratch to let `operator-sdk` scaffold the new project.
+2. Copy your existing code and configuration into the new project structure.
+
+**Note:** It is recommended that you have your project upgraded to the latest SDK release version (0.19.x+) before following the steps in this guide to migrate to new layout. However, see that might work from previous versions as well. In this case, if you found any issue which is not covered here then, check the previous [Migration Guides][migration-doc] which might help out.
```

Comment:
This section is redundant, since the `How to migrate` section is concise and says the same thing. The note should be moved up to the `Overview` section.
```suggestion

```

Reply:
I am not sure too. See that it needs to be inside after the ## How to migrate otherwise make no sense.
However, I think we can add as a subdomain of `## How to migrate`

Reply:
I'd like to see this section removed and the note moved up.